.github/workflows/ndc-nodejs-lambda-connector.yaml

```diff
@@ -0,0 +1,136 @@
+name: "ndc-nodejs-lambda connector"
+on:
+  pull_request:
+    branches:
+      - main
+      - test-ci/**
+  push:
+    branches:
+      - 'main'
+      - test-ci/**
+    tags:
+      - v**
+
+env:
+  DOCKER_REGISTRY: ghcr.io
+  DOCKER_IMAGE_NAME: hasura/ndc-nodejs-lambda
+
+jobs:
+  build-npm:
+    name: Build ndc-lambda-sdk npm package
+    defaults:
+      run:
+        working-directory: ./ndc-lambda-sdk
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          registry-url: https://registry.npmjs.org
+          cache: npm
+          cache-dependency-path: ./ndc-lambda-sdk/package-lock.json
+      - run: npm ci
+      - run: npm run build
+      - run: npm test
+
+  publish-npm:
+    name: Publish ndc-lambda-sdk to npm
+    defaults:
+      run:
+        working-directory: ./ndc-lambda-sdk
+    needs: build-npm
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          registry-url: https://registry.npmjs.org
+          cache: npm
+          cache-dependency-path: ./ndc-lambda-sdk/package-lock.json
+      - run: |
+          PACKAGE_VERSION=`npm version | sed -rn "2 s/.*: '([^']*)'.*/\1/g; 2 p"`
```

Comment:
wild sed expression there's no way to just output the info directly is there?

Reply:
Haha, you tell me, this is your sed expression, I don't know how that works. 😉😂  It's just moved here from the old build.
Personally I'd probably have just used jq on the `package.json` but why change what ain't broke?